Assets/Scripts/Controllers/TileSpriteController.cs

```diff
@@ -75,8 +80,20 @@ protected override void OnChanged(Tile tile)
             Debug.ULogErrorChannel("TileSpriteController", "tileGameObjectMap's returned GameObject is null -- did you forget to add the tile to the dictionary? Or maybe forget to unregister a callback?");
             return;
         }
-        
-        tile_go.GetComponent<SpriteRenderer>().sprite = SpriteManager.current.GetSprite("Tile", tile.Type.Name);
+
+        //TODO Evaluate this criteria and naming schema!
```

Comment:
Can we evaluate it before we merge the PR? Doesn't sound like a discussion that would be controversial.